articles/azure-vmware/configure-storage-policy.md

```diff
@@ -72,12 +72,12 @@ You'll run the `Get-StoragePolicy` cmdlet to list the vSAN based storage policie
 
 ## Set storage policy on VM
 
-You'll run the `Set-AvsVMStoragePolicy` cmdlet to Modify vSAN based storage policies on an individual VM. 
+You'll run the `Set-VMStoragePolicy` cmdlet to Modify vSAN based storage policies on an individual VM or on a group of VMs sharing a similar VM name. For example, if you have 3 VMs named "MyVM1", "MyVM2", "MyVM3", supplying "MyVM*" to the VMName parameter would change the StoragePolicy on all three VMs.
 
 >[!NOTE]
```

Comment:
@thinkdavid the Notes aren't showing correctly. Please update adding space after ">" Ex: > [!NOTE] Add a space after the ">" for second line too. Do this for lines 77, 78, 98, and 99.

Reply:
I think I did it, is this what you meant? @suzizuber